modules/gitops-release-notes-1-9-1.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-9-1_{context}"]
+= Release notes for {gitops-title} 1.9.1
+
+{gitops-title} 1.9.1 is now available on {product-title} 4.12 and 4.13.
+
+[id="errata-updates-1-9-1_{context}"]
+== Errata updates
+
+=== RHSA-2023:3591- {gitops-title} 1.9.1 security update advisory 
```

Comment:
is this the correct RHSA #? I tried going to it and it's giving me a security advisory for python3

Reply:
I saw that too, but i just got what was provided.... can we please get confirmation.

Reply:
@iam-veeramalla, @reginapizza , @varshab1210:

- For 1.9.1, Please confirm that it is ok to add [RHSA-2023:3591](https://access.redhat.com/errata/RHSA-2023:3591) - 4-digit ID in the Errata Updates section of the RN.
- Are we including this also - https://errata.engineering.redhat.com/advisory/116282?
For [116282](https://errata.engineering.redhat.com/advisory/116282), the only safe option I can see now is that wait till the errata state is changed to SHIPPED LIVE, and then 
@masauso-lungu check the errata advisory history and title for the changed number. If it is changed then please update the RN Errata updates section correctly, get the QE approval, and then send the RN PR for the merge.

@iam-veeramalla, @reginapizza , @varshab1210: WDYT? Please let me know. Thanks!

Reply:
Noted!